CHANGELOG.md

```diff
@@ -11,6 +11,52 @@ Some experimental features can be enabled / disabled using [Feature Gates](docs/
 
 ## Unreleased
 
+## 0.12.0 - 2020-12-22
+
+Includes all the changes from [0.11.1].
+
+### Added
+
+- Add support for rule-level AppliedTo for Antrea-native policies. ([#1396](https://github.com/vmware-tanzu/antrea/pull/1396), [@Dyanngg])
+  * Ability to select different endpoints on which to apply the different rules within the same policy, without having to define multiple policies
+  * For a given policy, either the policy-level AppliedTo field must be used, or the rule-level AppliedTo fields
+- Add support for port ranges in the rules of Antrea-native policies. ([#1557](https://github.com/vmware-tanzu/antrea/pull/1557), [@GraysonWu])
+- Introduce the FlowAggregator, an [IPFIX mediator] implementation to collect, process and export flow records generated by the Antrea Agents. ([#1671](https://github.com/vmware-tanzu/antrea/pull/1671) [#1677](https://github.com/vmware-tanzu/antrea/pull/1677), [@srikartati] [@dreamtalen] [@zyiou])
+  * Built using the [go-ipfix] library
+  * Processed flow records exported by the FlowAggregator are not missing any K8s contextual information (e.g. source / destination Pod names)
+  * It is recommended to always deploy the FlowAggregator when using the FlowExporter feature, as opposed to sending records directly from the Agent to a third-party collector
+  * Refer to the [Flow Exporter] documentation for more information
+- Add ability to sort by "effective priority" when listing internal NetworkPolicy resources (computed by the Controller) with antctl: priorities are sorted in the effective order in which they are enforced. ([#1530](https://github.com/vmware-tanzu/antrea/pull/1530), [@Dyanngg])
+- Add support for IPv6 to the FlowExporter implementation in the Agent. ([#1677](https://github.com/vmware-tanzu/antrea/pull/1677), [@lzhecheng] [@antoninbas] [@srikartati])
+  * Support for IPv6 IPFIX Information Elements in exported flow records
+  * Agent can export flows to an IPFIX collector over IPv6
+  * FlowAggregator is still missing support for IPv6
+- Add new "--k8s-1.15" flag to the  hack/generate-manifest.sh script to enable users to generate an Antrea manifest which is compatible with K8s 1.15 clusters (by default, Antrea requires K8s >= 1.16). ([#1664](https://github.com/vmware-tanzu/antrea/pull/1664), [@guesslin])
+
+### Changed
+
+- Update the priority of the default Tiers, to space them out more evenly and to provide more room for user-defined Tiers with higher priority than Emergency. ([#1665](https://github.com/vmware-tanzu/antrea/pull/1665), [@abhiraut])
+  * This change will impact users who use custom Tiers - in addition to the default Tiers -, as the relative priorities between tiers may change and impact the order in which Antrea-native policies are enforced
+  * Impacted users will need to update the priority values of their custom tiers after upgrading Antrea to restore the enforcement order of their policies
```

Comment:
Since we technically don't allow Tier updates.. something on the lines of
```suggestion
  * Impacted users will need to recreate their custom tiers with updated priority values after upgrading Antrea to restore the enforcement order of their policies
```

Reply:
Thanks, will address before I merge